tests/_utils/view.js

```diff
@@ -0,0 +1,836 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* jshint latedef:false */
```

Comment:
No, no – we should remove (or add) that option to JSHint config. It's totally natural that you want the code of tests to be located at the front and AFAIS class definition works like a function definition (worth confirming with the spec or some article). So the same way we define functions at the end of the scope we should define the classes as well.